net/mscp/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                mscp
+version             0.2.1
+categories          net
+platforms           darwin
+license             GPL-3.0
+maintainers         @lasmarois \
+                    nicomarois.com:mail
+description         multi-threaded scp
+long_description    mscp, a variant of scp, copies files over multiple SSH (SFTP) connections by multiple threads.
+homepage            https://github.com/upa/mscp
+master_sites        https://github.com/upa/mscp/releases/download/v{version}
+
+checksums           rmd160  2901efb16a5b001a8bdef924fc366622244b67c0 \
+                    sha256  ea9aa82d3c055285705e39cbf1af3f02de33b6a1c767fc06f6dc91f36ddba4bc \
+                    size    900419
+
+# CMake configuration
+configure.args --install-prefix ${prefix} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_C_COMPILER=${configure.cc} \
+    -DCMAKE_CXX_COMPILER=${configure.cxx}
+
+use_configure       yes
```

Comment:
`use_configure yes` is the implicit default, especially if using the `cmake` portgroup.

This line can be deleted.